operator/src/main/helm/coherence-operator/templates/deployment.yaml

```diff
@@ -81,15 +81,23 @@ spec:
 {{- if .Values.logCaptureEnabled }}
             - name: EFK_INTEGRATION_ENABLED
               value: {{ .Values.logCaptureEnabled | quote }}
-  {{- if .Values.elasticsearch }}
+  {{- if .Values.elasticsearchEndpoint }}
     {{- if .Values.elasticsearchEndpoint.host }}
             - name: ES_HOST
-              value: {{ .Values.elasticsearchEndpoint.host }}
+              value: {{ .Values.elasticsearchEndpoint.host | quote }}
     {{- end }}
     {{- if .Values.elasticsearchEndpoint.port }}
             - name: ES_PORT
               value: {{ .Values.elasticsearchEndpoint.port | quote }}
     {{- end }}
+    {{- if .Values.elasticsearchEndpoint.user }}
+            - name: ES_USER
+              value: {{ .Values.elasticsearchEndpoint.user | quote }}
+    {{- end }}
+    {{- if .Values.elasticsearchEndpoint.password }}
+            - name: ES_PASSWORD
+              value: {{ .Values.elasticsearchEndpoint.password | quote }}
+    {{- end }}
```

Comment:
Why are we specifying ES-related env vars directly from the values file for the operator container, and from the secret for the fluentd container? I think we should be using secret for both, if we even need any of these values for the operator container (do we?)

We should also standardize on env var names to use: why are we using `ES_*` for the operator and `ELASTICSEARCH_*` for fluentd?